tests/Monolog/Processors/CommonProcessorTest.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+namespace tests\Monolog\Processors;
+
+use Symfony\Component\HttpFoundation\Request;
```

Comment:
Please do an optimize imports in PHPUnit, it organises imports alphabetically.

Reply:
in PHPStorm, right @jkrnak ?